mw-webapp/src/service/DayReportService.ts

```diff
@@ -18,14 +27,32 @@ const QUERY_LIMIT = 30;
  */
 export type DayReportDTOWithoutUuid = Omit<DayReportDTO, "uuid">;
 
+/**
+ * Parse and validate json
+ */
+const parseStringifiedModel = <T extends z.ZodTypeAny>(content: string, schema: T) => {
```

Comment:
Let's use words instead of letters (T)
Looks like this function validates data,

content - rawData
parseStringifiedModel - parseWithValidation

Reply:
Changed , thanks